taxi/urls.py

```diff
@@ -7,17 +7,61 @@
     DriverListView,
     DriverDetailView,
     ManufacturerListView,
+    CarCreateView,
+    CarUpdateView,
+    CarDeleteView,
+    ManufacturerCreateView,
+    ManufacturerUpdateView,
+    ManufacturerDeleteView,
 )
 
-urlpatterns = [
-    path("", index, name="index"),
+cars_urlpatterns = [
+    path("", CarListView.as_view(), name="car-list"),
+    path("create/", CarCreateView.as_view(), name="car-create"),
     path(
-        "manufacturers/",
+        "<int:pk>/",
+        include(
+            [
+                path("", CarDetailView.as_view(), name="car-detail"),
+                path("update/", CarUpdateView.as_view(), name="car-update"),
+                path("delete/", CarDeleteView.as_view(), name="car-delete"),
+            ]
+        ),
+    ),
+]
+
+manufacturers_urlspatterns = [
+    path(
+        "",
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
-    path("cars/", CarListView.as_view(), name="car-list"),
-    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
+    path(
+        "create/", ManufacturerCreateView.as_view(), name="manufacturer-create"
+    ),
+    path(
+        "<int:pk>/",
+        include(
+            [
+                path(
+                    "update/",
+                    ManufacturerUpdateView.as_view(),
+                    name="manufacturer-update",
+                ),
+                path(
+                    "delete/",
+                    ManufacturerDeleteView.as_view(),
+                    name="manufacturer-delete",
+                ),
+            ]
+        ),
+    ),
+]
+
```

Comment:
is it easy to read this code ?

Reply:
I've followed practice explained [here](https://docs.djangoproject.com/en/4.2/topics/http/urls/#including-other-urlconfs).